rfcs/load_test.md

```diff
@@ -0,0 +1,45 @@
+# RFC 33: Load Tests
+
+## Summary
+
+Add a test type in which the test document is simply loaded. The test
+passes if this occurs without encountering a fatal error e.g. a crash.
+
+## Details
+
+Load Tests are static tests typically used for cases where browsers
+are known to have a bug such as a crash or a leak. They differ from
+other tests such as reftests and testharness tests in that there is no
+pass condition other than the page loading successfully. In vendor
+infrastructure these tests may also be run with additional
+instrumentation e.g. address sanitizing, making the pass condition
+more stringent. The lack of an explicit pass condition makes it easier
+to write these tests than tests where a detailed understanding of the
+correct behaviour is required, making them suitable for autogeneration
+e.g. as the output of a fuzzing tool.
+
+The infrastructure changes are as follows:
+
+* A new test type `load` recognised by the manifest and assigned to
+  any test file with a name containing `-load` or `-loadtest`
```

Comment:
`-load` is too generic and is already used by existing tests e.g.
```
./idle-detection/idle-detection-allowed-by-feature-policy-attribute-redirect-on-load.https.sub.html
./css/css-paint-api/valid-image-after-load.https.html
```
`-loadtest` suffix or `loadtests` directory seems OK.

Reply:
Since "load" can be taken to mean "encumber," people may expect these are stress tests. @zcorpan mentioned that "crash" is already conventional, so maybe that represents a preference in terminology. Would there be a problem with updating those tests?